dev/ansible/01_generate_secrets_vault.yml

```diff
@@ -0,0 +1,18 @@
+- name: Generate Secrets with Vault
+  shell: |
+      vault kv put secret/postgres \
+      username=$(vault write -field='value' gen/password length=12 symbols=0) \
+      password=$(vault write -field='value' gen/password length=12 symbols=0)
+  run_once: true
+  environment:
+      VAULT_ADDR: "http://127.0.0.1:8200"
+
+- name: Register username key
+  shell:
+      cmd: vault kv get -field='username' secret/postgres
+  register: username
+
+- name: Register password key
+  shell:
+      cmd: vault kv get -field='password' secret/postgres
+  register: password
```

Comment:
You don't need this for anything, do you?
```suggestion
```

Reply:
No, that's right!